pkg/loop/internal/test/types/interfaces.go

```diff
@@ -75,6 +75,12 @@ type ErrorLogEvaluator interface {
 	Evaluator[types.ErrorLog]
 }
 
+// ValidationEvaluator is a helper interface for testing ErrorLogs
```

Comment:
@george-dorin I think your comment is mistaken here :) ErrorLogs -> ValidationService?

Reply:
Fixed in [`75ca4bb` (#402)](https://github.com/smartcontractkit/chainlink-common/pull/402/commits/75ca4bb87e3b7d6a714ebda797351f090c972082)